src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js

```diff
@@ -7,11 +7,13 @@
 	 * @param ParameterService for passing parameter information between modal and page
 	 * @constructor
 	 */
-	function PipelineController($scope, $http, CartService, ParameterService) {
+	function PipelineController($scope, $http, CartService, ParameterService, ToolDataTableService) {
```

Comment:
`DataTables` is a javascript library we use on a lot of the pages in IRIDA.  Can you rename anything to do with the Galaxy Data Table to be specificly `GalaxyDataTableService` or something similar, just to make it more clear.